subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/NonHierarchicalFileWatcherUpdater.java

```diff
@@ -158,10 +142,9 @@ private void updateWatchedDirectories(Map<String, Integer> changedWatchDirectori
                 }
             }
         });
-        if (watchedDirectories.isEmpty()) {
-            LOGGER.info("Not watching anything anymore");
-        }
+
         LOGGER.info("Watching {} directories to track changes", watchedDirectories.entrySet().size());
```

Comment:
I don't think it is very useful to log this every time we change the number of watched directories on Linux. This will change quite often and create quite a bit of logs.

Reply:
I share the feeling, though we had this here for a long time. Let's move it in a separate PR.